.github/workflows/macos_publish_dmg_release.yml

```diff
@@ -402,7 +402,7 @@ jobs:
     # Run if release-type is provided (not empty) an is not internal
     if: github.event.inputs.release-type != null && github.event.inputs.release-type != 'internal'
 
-    uses: duckduckgo/macos-browser/.github/workflows/create_variants.yml@main
+    uses: duckduckgo/macos-browser/.github/workflows/macos_create_variants.yml@main
```

Comment:
This is the one tricky part - this workflow is always called from main branch. And the calling workflow (`publish_dmg_release.yml`), when run from the release branch next Monday (for public release), would still look for `create_variants.yml` in main branch, won't find it and will fail. This isn't critical, because we can run create_variants.yml manually (it's the last step of the workflow), but we could also fix it by changing this very line in the release branch. This way the release branch workflow would know the new name of the main branch workflow.

Reply:
Ok since this is a release branch PR, the above won't be an issue, let's just merge the release branch to main right after merging this PR. Sorry for the confusion :)

Reply:
Thanks for confirming, I had the same thought that this would be a problem if the branches stayed out of sync for long, so I expected to manually kick off the internal bump after merging. Which will happen momentarily!